### PR TITLE
Add release notes for 2.0.3

### PR DESCRIPTION
**Pull request by elefeint**
I'll copy those into the github release once merged.

### CODE REVIEW

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -5,6 +5,28 @@ https://spring.io/projects/spring-cloud-gcp[Spring Cloud GCP] is a set of integr
 This document provides a high-level overview of the changes introduced in Spring Cloud GCP by release.
 For a detailed view of what has changed, refer to the https://github.com/GoogleCloudPlatform/spring-cloud-gcp/commits/main[commit history] on GitHub.
 
+== 2.0.3 (2021-06-08)
```
Comment (meltsufin) [inline CHANGELOG.adoc:8]:
```suggestion
== 2.0.3 (2021-06-09)
```

File: CHANGELOG.adoc [hunk 2]
```diff
@@ -5,6 +5,28 @@ https://spring.io/projects/spring-cloud-gcp[Spring Cloud GCP] is a set of integr
...
+
+=== Cloud Spanner
+* Stopped Cloud Spanner emulator autoconfiguration from triggering default credentials creation (https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/457[#457]).
+* Added ability to customize arbitrary `SpannerOptions` settings by configuring a `SpannerOptionsCustomizer` bean (https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/489[#489]).
+
+=== Cloud SQL
+* Added IAM authentication option for PostgreSQL (https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/488[#488]).
+* Enabled placeholder interpretation in Cloud SQL properties (https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/495[#495]).
+
+=== Notable version changes
```
Comment (meltsufin) [inline CHANGELOG.adoc:24]: I usually put this kind of stuff at the top under "General".

File: CHANGELOG.adoc [hunk 3]
```diff
@@ -5,6 +5,28 @@ https://spring.io/projects/spring-cloud-gcp[Spring Cloud GCP] is a set of integr
...
+* Upgraded to libraries BOM 20.6.0
+* Added version tracking for `com.google.cloud.sql:jdbc-socket-factory-core` (https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/466[#466])
```
Comment (meltsufin) [inline CHANGELOG.adoc:26]:
```suggestion
* Added version management for `com.google.cloud.sql:jdbc-socket-factory-core` (https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/466[#466])
```

File: CHANGELOG.adoc [hunk 4]
```diff
@@ -5,6 +5,28 @@ https://spring.io/projects/spring-cloud-gcp[Spring Cloud GCP] is a set of integr
 This document provides a high-level overview of the changes introduced in Spring Cloud GCP by release.
 For a detailed view of what has changed, refer to the https://github.com/GoogleCloudPlatform/spring-cloud-gcp/commits/main[commit history] on GitHub.
 
+== 2.0.3 (2021-06-08)
+
+=== Cloud Pub/Sub
+* Exposed publisher endpoint for message ordering (https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/421[#421])
+* Pub/Sub Health Indicator timeout increased to 2 seconds (https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/420[#420]).
+* Gated Cloud Pub/Sub emulator autoconfiguration Pub/Sub module being present and enabled (https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/446[#446])
+* `PubSubMessageHandler` now passes the original message to new success/failure callbacks, allowing applications to track message publish status (https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/482[#482]).
```
Comment (meltsufin) [inline CHANGELOG.adoc:14]: Should we mention that `setPublishCallback()` is not deprecated?

**Review by meltsufin — CHANGES_REQUESTED** [review]

**Comment by meltsufin** [comment]
> I'll copy those into the github release once merged.

In the previous releases, I just linked to the changelog to avoid duplication.

File: CHANGELOG.adoc (hunk 1 above)
Comment (elefeint) [reply CHANGELOG.adoc:8]: Interesting question; technically the release jars went out yesterday.

File: CHANGELOG.adoc (hunk 4 above)
Comment (elefeint) [reply CHANGELOG.adoc:14]: yep.

File: CHANGELOG.adoc (hunk 1 above)
Comment (meltsufin) [reply CHANGELOG.adoc:8]: I didn't realize, I take that back.

**Review by meltsufin — APPROVED** [review]

**Pull request merged**